x-pack/plugins/cloud_security_posture/common/types/rules/v3.ts

```diff
@@ -8,7 +8,7 @@
 import { schema, TypeOf } from '@kbn/config-schema';
 import { CSPM_POLICY_TEMPLATE, KSPM_POLICY_TEMPLATE } from '../../constants';
 
-const DEFAULT_BENCHMARK_RULES_PER_PAGE = 25;
+export const DEFAULT_BENCHMARK_RULES_PER_PAGE = 25;
```

Comment:
sorry, i think i wasn't clear enough, this constant is sitting in a types folder, and its versioned for some reason. its not a type and it doesnt need to be versioned. i know this line is not new but i still find it odd. can you move this const to the rules table page somewhere?